shell/platform/fuchsia/flutter/gfx_external_view_embedder.h

```diff
@@ -172,11 +178,23 @@ class GfxExternalViewEmbedder final : public flutter::ExternalViewEmbedder {
     bool pending_focusable = true;
   };
 
-  struct ScenicLayer {
+  struct ScenicImage {
```

Comment:
Can we document this struct and ScenicLayer?

Reply:
Done